.github/workflows/test-radosgw.yml

```diff
@@ -35,15 +38,40 @@ jobs:
           driver-opts: |
             network=host
 
+      - name: Download Radogw Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ needs.build.outputs.artifact_id }}
+
+      - name: Unpack Artifacts
+        run: |
+          tar -xvf build-results.tar.gz
+
+      - name: Build S3GW Container
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          tags: |
+            localhost:5000/s3gw:latest
+          file: s3gw-core/build/Dockerfile.build-container
+          context: ceph/build
+
+      - name: Install S3cmd
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y s3cmd
+
       - name: Run Tests
         run: |
           mkdir storage
-          docker run \
-            -v ./storage:/data \
+          docker run -d \
+            -v $GITHUB_WORKSPACE/storage:/data \
             -p 7480:7480 \
-            mrohrich/s3gw:${{ needs.call-build-workflow.outputs.short_sha }}
+            localhost:5000/s3gw:latest \
+              --rgw-backend-store simplefile \
+              --debug-rgw 1
 
+          sleep 5  # This is needed to give the gateway some time to boot up.
```

Comment:
Do you think is it possible to rely on something different and more robust than a sleep to check for s3gw readiness?
Eg: s3cmd ls command inside a loop?
Btw, a sleep can be ok on a first approximation.

Reply:
Yes, you're right a sleep isn't very robust. Since long startup times need special attention in K8s though (wrt. liveness/readiness probes), this serves dual purpose: 1) give the gateway *some* startup time and 2) if the gateway takes too long to boot, the tests will fail, alerting us that the startup time has increased sufficiently to matter to the K8s probes.
I should probably add that reasoning to the comment, but otherwise I'd like to merge it as is.

Reply:
Ok, if this is intended as a check to probe that the process _must_ be ready within 5 seconds, then ok, this is perfectly fine.
Yes, probably a comment explaining this would be nice.